lib/tasks/trn_generator.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class TRNGenerator
+  class << self
+    def next
+      available.pop || (raise "TRN available list exhausted")
+    end
+
+  private
+
+    ALL_TRNS = ("0011111".."9999999").to_a.freeze unless defined?(ALL_TRNS)
```

Comment:
Soooo... Will this just load 10 milion strings into memory when this class is loaded?

Can we test how it works on a review app?

Reply:
Yes. That's the point of it having so many. I wanted to make sure that the review app could cope with them all if we for some reason wanted to generate all possibly valid 10M records. I think that it'll be unlikely that we need more than 200k at any time, so the next PR will restrict this for the rake task.